code/game/gamemodes/changeling/powers/fleshmend.dm

```diff
@@ -1,25 +1,41 @@
 /obj/effect/proc_holder/changeling/fleshmend
 	name = "Fleshmend"
-	desc = "Our flesh rapidly regenerates, healing our wounds."
-	helptext = "Heals a moderate amount of damage over a short period of time. Can be used while unconscious."
+	desc = "Our flesh rapidly regenerates, healing our wounds. Effectiveness decreases with quick, repeated use."
+	helptext = "Heals a moderate amount of damage over a short period of time. Can be used while unconscious, and will alert nearby crew."
 	chemical_cost = 25
 	dna_cost = 2
 	req_stat = UNCONSCIOUS
+	var/recent_uses = 1 //The factor of which the healing should be divided by
+
+/obj/effect/proc_holder/changeling/fleshmend/New()
+	..()
+	SSobj.processing.Add(src)
+
+/obj/effect/proc_holder/changeling/fleshmend/Destroy()
+	SSobj.processing.Remove(src)
+	..()
+
+/obj/effect/proc_holder/changeling/fleshmend/process()
+	if(recent_uses > 1)
+		spawn(20) //Around 3 seconds between each use decrease
+			recent_uses--

```

Comment:
This is bad
With this it is possible to make this a zero since when recent uses is 2 it spawns without stopping the global loop, new loop comes faster than the spawn expires, passing the check and setting the spawn to decrease the var again, in the end when both spawns expire you will go to 0 and it will runtime like shit since you divide by this var in the action proc